docs/about.md

```diff
@@ -0,0 +1,18 @@
+# About the Project
+
+The Python scientific stack, compiled to WebAssembly.
+
+Pyodide brings the Python runtime to the browser via WebAssembly, along with
+the Python scientific stack including NumPy, Pandas, Matplotlib, parts of
+SciPy, and NetworkX. The [packages
+directory](https://github.com/iodide-project/pyodide/tree/master/packages)
+lists over 35 packages which are currently available.
```

Comment:
This copies the about section while previously it was linked from the readme. The issue with the previous approach is that relative links can't work both in the documentation and the readme and TOC rendering was not great https://pyodide.readthedocs.io/en/latest/rootdir.html

This approach means these 3 paragraphs need to be maintained in sync between the doc and the readme but we have more flexibility.